scripts/manta_prepare_sv_vcf_f94bcc1.py

```diff
@@ -0,0 +1,1670 @@
+    :param res: 4 code number for the current type of read orientation
+    :param lrnoise: ordered list representing the count of reads considered as noise
+    :param aln_rec:
+    :param svlen: value only use in the logger; will be deprecated in later version
+    :param insert_size: mean insert size
+    :param sigma: sigma can be a sigma value or the standard deviation
+    :param factor_of_sigma: if sigma value represents a standard deviation, this factor must be a value of 1
+    :return:
+    """
+    try:
+        a = aln_rec
+        if (res == '1100' or res == '1001') and (a.reference_name != a.next_reference_name):
+            logger.debug("TRANSLOCATION READ: " + str(str(aln_rec).split("\t")[0:8]))
+            lrnoise[4] += 1  # TRA
+            lrnoise[5] += 1  # TOT
+        elif (res == '1100' or res == '1001') and (a.reference_name == a.next_reference_name):
+            logger.debug("DUPLICATION READ: " + str(str(aln_rec).split("\t")[0:8]))
+            lrnoise[1] += 1  # DUP
+            lrnoise[5] += 1  # TOT
+        elif res == '1110' or res == '1011' or res == '0100' or res == '0001':
+            if a.reference_name != a.next_reference_name:
+                logger.debug("TRANSLOCATION READ: " + str(str(aln_rec).split("\t")[0:8]))
+                lrnoise[4] += 1  # TRA
+            else:
+                logger.debug("INVERSION READ " + str(str(aln_rec).split("\t")[0:8]))
+                lrnoise[3] += 1  # INV
+            lrnoise[5] += 1  # TOT
+        elif res == '0110' or res == '0011':
+            logger.debug("svlen = " + str(svlen) + " and INDEL isize = " + str(a.isize))
+            if a.reference_name != a.next_reference_name:
+                logger.debug("TRANSLOCATION READ: " + str(str(aln_rec).split("\t")[0:8]))
+                lrnoise[4] += 1  # TRA
+                lrnoise[5] += 1  # TOT
+            elif a.isize < (insert_size - (factor_of_sigma * sigma)) or a.isize > insert_size + (factor_of_sigma * sigma):
+                lrnoise[0] += 1  # DEL
+                logger.debug("DELETION READ: " + str(str(aln_rec).split("\t")[0:8]))
+                lrnoise[5] += 1  # TOT
+            elif a.isize < (insert_size - (factor_of_sigma * sigma)) or a.isize > insert_size + (factor_of_sigma * sigma):
+                lrnoise[2] += 1  # INS
+                logger.debug("INSERTION READ: " + str(str(aln_rec).split("\t")[0:8]))
+                lrnoise[5] += 1  # TOT
+        else:
+            logger.error("res_code: {}".format(str(res)))
+            logger.error("No code assigned to the res object: Should Not have happened; Check with Authors")
+    except Exception as e:
+        logger.exception(e)
+        exit(2)
+    return lrnoise
+
+
+def info_on_captured_discordant_reads(bamh, slop, chrom, pos):
+    """
+    FUNCTION NOT USE ; Was for testing only ; info_on_captured_discordant_reads
```

Comment:
NOT USE? Is it require for this PR?

Reply:
Yes it is required. These scripts are taken directly from TGEN. We're taking them as is so pay no mind to the content.